python/pyspark/ml/regression.py

```diff
@@ -619,6 +627,22 @@ def getSubsamplingRate(self):
         """
         return self.getOrDefault(self.subsamplingRate)
 
+    @since("1.4.0")
+    def setFeatureSubsetStrategy(self, value):
+        """
+        Sets the value of :py:attr:`featureSubsetStrategy`.
+
+        .. note:: Deprecated in 2.1.0 and will be removed in 3.0.0.
```

Comment:
This should technically be marked as deprecated in 2.4.0, even though the Scala version was before

Reply:
Fixed. Thanks!